README.md

```diff
@@ -23,10 +23,10 @@ input and returns an object with two properties:
 Below is a list of rules that valid `npm` package name should conform to.
 
 - package name length should be greater than zero
-- all the characters in the package name must be lowercase i.e., no uppercase or mixed case names are allowed
+- package name should not be start from a special character.
+- package name should not have any uppercase letter.
 - package name *can* consist of hyphens
 - package name must *not* contain any non-url-safe characters (since name ends up being part of a URL)
-- package name should not start with `.` or `_`
 - package name should *not* contain any leading or trailing spaces
 - package name should *not* contain any of the following characters: `~)('!*`
 - package name *cannot* be the same as a node.js/io.js core module nor a reserved/blacklisted name. For example, the following names are invalid:
```

Comment:
Update the examples section below.

Reply:
Will do it once I get the confirmation from Stefan regarding the "@" at the beginning of package name.

Reply:
No need to update the examples now.